CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Breaking changes:
 
 Features:
 
+- [#1248](https://github.com/rails-api/active_model_serializers/pull/1248) Add support for JSON API deserialization (@beauby)
```

Comment:
Also, If I'm reading this code right, this isn't integrated anywhere, so maybe it would be to mark it as an experimental feature so we can merge it and iterate faster?